### PR TITLE
More translations for ja.yml

### PR DESCRIPTION
**Pull request by matcha440**
# Description 
This round includes reports and notifications.

## Corresponding Issue
#1215 

# Test Coverage

✅

### CODE REVIEW

File: config/locales/ja.yml [hunk 1]
```diff
@@ -77,21 +77,21 @@ ja:
...
-    reporter: Reporter
-    user_banned: '%{name} has been banned'
-    ban_removed: 'Ban removed on %{name}'
-    user_banned_error: 'Could not ban %{name}'
-    ban_removed_error: 'Could not remove ban on %{name}'
+    admin_dashboard: '管理者ダッシュボード'
+    reported_users: '報告されたユーザー'
+    banned_users: 'アクセス禁止のユーザー'
+    reasons: 理由
+    new: '%{name}新しい報告'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
    new: '%{name}さんの新しい報告'
```

File: config/locales/ja.yml [hunk 2]
```diff
@@ -77,21 +77,21 @@ ja:
...
+    remove_ban: 'アクセス禁止を外します。'
+    ban_user: 'ユーザーをアクセス禁止にします'
+    report_sent: '%{name}の報告が送られました。'
+    report_error: '%{name}の報告を送ることができませんでした。'
+    reportee: 報告されたユーザー
+    reporter: 報告したユーザー
+    user_banned: '%{name}がアクセス禁止されました'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
    user_banned: '%{name}さんがアクセス禁止にされました'
```

File: config/locales/ja.yml [hunk 3]
```diff
@@ -77,21 +77,21 @@ ja:
...
+    ban_removed: '%{name}のアクセス禁止を外しました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
    ban_removed: '%{name}さんのアクセス禁止を外しました。'
```

File: config/locales/ja.yml [hunk 4]
```diff
@@ -77,21 +77,21 @@ ja:
...
+    user_banned_error: '%{name}をアクセス禁止にすることができませんでした'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
    user_banned_error: '%{name}さんをアクセス禁止にすることができませんでした'
```

File: config/locales/ja.yml [hunk 5]
```diff
@@ -77,21 +77,21 @@ ja:
...
+    ban_removed_error: '%{name}のアクセス禁止を外すことができませんでした'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
    ban_removed_error: '%{name}さんのアクセス禁止を外すことができませんでした'
```

File: config/locales/ja.yml [hunk 6]
```diff
@@ -306,62 +306,62 @@ ja:
...
+    clear: クリア
+    created_by_html: '<strong>作成者</strong>：%{name}'
+    preview: 'もっと見る'
     members:
-      title: 'List of Members'
+      title: 'メンバーのリスト'
     ally:
-      accepted: '%{name} accepted your ally request!'
-      sent_html: '%{link_to_user} sent an ally request!'
+      accepted: '%{name}はあなたの仲間リクエストを承認しました！'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      accepted: '%{name}さんがあなたの仲間リクエストを承認しました！'
```

File: config/locales/ja.yml [hunk 7]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      sent_html: '%{link_to_user}は仲間リクエストを送りました！'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      sent_html: '%{link_to_user}さんがあなたに仲間リクエストを送りました！'
```

File: config/locales/ja.yml [hunk 8]
```diff
@@ -306,62 +306,62 @@ ja:
...
     comment:
-      full: '%{name} commented "%{comment}" on %{typename}'
-      truncated: '%{name} commented "%{comment}" [...] on %{typename}'
+      full: '%{name}は%{typename}について「%{comment}」をコメントしました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      full: '%{name}さんが%{typename}について「%{comment}」をコメントしました。'
```

File: config/locales/ja.yml [hunk 9]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      truncated: '%{name}は%{typename}について「%{comment}」 [...]をコメントしました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      truncated: '%{name}さんが%{typename}について「%{comment}」 [...]をコメントしました。'
```

File: config/locales/ja.yml [hunk 10]
```diff
@@ -306,62 +306,62 @@ ja:
...
     group:
-      new_group: '%{name} created a group "%{group_name}"'
-      new_group_member: '%{name} joined your group "%{group_name}"'
-      add_group_leader: '%{name} became a leader of "%{group_name}"'
-      remove_group_leader: '%{name} is no longer a leader of "%{group_name}"'
+      new_group: '%{name}は%{group_name}というグループを作りました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      new_group: '%{name}さんが%{group_name}というグループを作りました。'
```

File: config/locales/ja.yml [hunk 11]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      new_group_member: '%{name}はあなたのグループ%{group_name}に入りました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      new_group_member: '%{name}さんがあなたのグループ%{group_name}に入りました。'
```

File: config/locales/ja.yml [hunk 12]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      add_group_leader: '%{name}は%{group_name}のリーダーになりました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      add_group_leader: '%{name}さんが%{group_name}のリーダーになりました。'
```

File: config/locales/ja.yml [hunk 13]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      remove_group_leader: '%{name}は今から%{group_name}のリーダーではありません。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      remove_group_leader: '%{name}さんはもう%{group_name}のリーダーではありません。'
```

File: config/locales/ja.yml [hunk 14]
```diff
@@ -306,62 +306,62 @@ ja:
...
     meeting:
-      new_meeting: '%{name} created a new meeting "%{meeting_name}" for "%{group_name}"'
-      remove_meeting: '%{name} has cancelled "%{meeting_name}" for "%{group_name}"'
-      update_meeting: '%{name} has updated "%{meeting_name}" for "%{group_name}"'
-      join_meeting: '%{name} has joined "%{meeting_name}" for "%{group_name}"'
+      new_meeting: '%{name}は%{group_name}で%{meeting_name}という会議を作成しました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      new_meeting: '%{name}さんが%{group_name}で%{meeting_name}という会議を作成しました。'
```

File: config/locales/ja.yml [hunk 15]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      remove_meeting: '%{name}は%{group_name}の%{meeting_name}という会議を取り止めました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      remove_meeting: '%{name}さんが%{group_name}の%{meeting_name}という会議を取り止めました。'
```

File: config/locales/ja.yml [hunk 16]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      update_meeting: '%{name}は%{group_name}の%{meeting_name}という会議を更新しました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      update_meeting: '%{name}さんが%{group_name}の%{meeting_name}という会議を更新しました。'
```

File: config/locales/ja.yml [hunk 17]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      join_meeting: '%{name}は%{group_name}の%{meeting_name}という会議に参加しました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      join_meeting: '%{name}さんが%{group_name}の%{meeting_name}という会議に参加しました。'
```

File: config/locales/ja.yml [hunk 18]
```diff
@@ -306,62 +306,62 @@ ja:
...
     mailer:
-      comment_on_moment_subject: 'if me | %{user} commented on your moment "%{moment}"'
-      comment_on_strategy_subject: 'if me | %{user} commented on your strategy "%{strategy}"'
-      comment_on_meeting_subject: 'if me | %{user} commented on the meeting "%{meeting}" in the group "%{group}"'
-      comment_on_private_body: '<p>Your ally <strong>%{user}</strong> commented privately:</p>'
-      comment_on_body: '<p>Your ally <strong>%{user}</strong> commented:</p>'
+      comment_on_moment_subject: 'if me | %{user}は%{moment}というモーメントについてコメントをしました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      comment_on_moment_subject: 'if me | %{user}さんが%{moment}というモーメントについてコメントをしました。'
```

File: config/locales/ja.yml [hunk 19]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      comment_on_strategy_subject: 'if me | %{user}は%{strategy}という作戦についてコメントをしました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      comment_on_strategy_subject: 'if me | %{user}さんが%{strategy}という作戦についてコメントをしました。'
```

File: config/locales/ja.yml [hunk 20]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      comment_on_meeting_subject: 'if me | %{user}は%{group_name}の%{meeting_name}という会議にコメントをしました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      comment_on_meeting_subject: 'if me | %{user}さんが%{group_name}の%{meeting_name}という会議にコメントをしました。'
```

File: config/locales/ja.yml [hunk 21]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      comment_on_private_body: '<p>あなたの仲間<strong>%{user}</strong>に非公開のコメントをしました：</p>'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      comment_on_private_body: '<p>あなたの仲間<strong>%{user}さん</strong>がプライベートコメントをしました：</p>'
```

File: config/locales/ja.yml [hunk 22]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      comment_on_body: '<p>あなたの仲間<strong>%{user}</strong>コメントをしました：</p>'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      comment_on_body: '<p>あなたの仲間<strong>%{user}さん</strong>がコメントをしました：</p>'
```

File: config/locales/ja.yml [hunk 23]
```diff
@@ -306,62 +306,62 @@ ja:
...
-      join_meeting_body: '<p>To see "%{meeting}", %{link}!</p>'
-      reported_user_subject: 'if me | You have reported a user'
-      reported_user_body: 'You have reported %{reported_name}. This will be reviewed by an admin.'
-      reportee_user_subject: 'if me | You have been reported'
-      reportee_user_body: 'You have been reported. This will be reviewed by an admin.'
-      add_ban_subject: 'if me | You have been banned'
-      add_ban_body: 'You have been banned for violating our %{code_of_conduct_link}.'
-      remove_ban_subject: 'if me | You are no longer banned'
-      remove_ban_body: 'You were previously banned for violating our %{code_of_conduct_link}. You are no longer banned.'
+      comment_link: '<p>読むために、%{link}!</p>'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      comment_link: '<p>読むにはこちらへ：%{link}!</p>'
```

File: config/locales/ja.yml [hunk 24]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      new_group_subject: 'if me | %{user}は%{group}というグループを作成しました。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      new_group_subject: 'if me | %{user}さんが%{group}というグループを作成しました。'
```

File: config/locales/ja.yml [hunk 25]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      new_group_body: '<p>%{user}は%{group}というグループを作成しました。</p><p><i>%{description}</i></p><p>詳しくは、%{link}!</p><p>%{link_to_code_of_conduct}に従ってください！</p>'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      new_group_body: '<p>%{user}さんが%{group}というグループを作成しました。</p><p><i>%{description}</i></p><p>詳しくは、%{link}!</p><p>%{link_to_code_of_conduct}に従ってください！</p>'
```

File: config/locales/ja.yml [hunk 26]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      new_group_member_subject: 'if me | %{user}は%{group}というグループに参加します'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      new_group_member_subject: 'if me | %{user}さんがあなたのグループ、%{group}に参加しました'
```

File: config/locales/ja.yml [hunk 27]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      add_group_leader_you_subject: 'if me | あなたは%{group}のリーダーになりました'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      add_group_leader_you_subject: 'if me | あなたが%{group}のリーダーになりました'
```

File: config/locales/ja.yml [hunk 28]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      add_group_leader_subject: 'if me | %{user}は%{group}のリーダーになりました'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      add_group_leader_subject: 'if me | %{user}さんが%{group}のリーダーになりました'
```

File: config/locales/ja.yml [hunk 29]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      add_remove_group_leader_body: '<p>%{group}を詳しく見るには、 %{link}！</p>'
+      remove_group_leader_you_subject: 'if me | あなたは%今から{group}のリーダーではありません'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      remove_group_leader_you_subject: 'if me | あなたはもう%{group}のリーダーではありません'
```

File: config/locales/ja.yml [hunk 30]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      remove_group_leader_subject: 'if me | %{user}は今から%{group}のリーダーではありません'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      remove_group_leader_subject: 'if me | %{user}さんはもう%{group}のリーダーではありません'
```

File: config/locales/ja.yml [hunk 31]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      update_meeting_subject: 'if me | %{user}は%{group}の%{meeting}という会議を更新しました'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      update_meeting_subject: 'if me | %{user}さんが%{group}の%{meeting}という会議を更新しました'
```

File: config/locales/ja.yml [hunk 32]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      new_meeting_subject: 'if me | %{user}は%{group}の%{meeting}という会議を作成しました'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      new_meeting_subject: 'if me | %{user}さんが%{group}の%{meeting}という会議を作成しました'
```

File: config/locales/ja.yml [hunk 33]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      meeting_body: '<p>%{subject}</p><p><i>%{description}</i></p><p><strong>場所：</strong>%{location}</p><p><strong>日付：</strong>%{date}</p><p><strong>時間：</strong>%{time}</p>'
+      new_meeting_link: '<p>詳しくは、%{link}！</p>'
+      update_meeting_link: '<p>詳しくは、%{link}！</p>'
+      remove_meeting_subject: 'if me | %{user}は%{group}の%{meeting}という会議をキャンセルしました'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      remove_meeting_subject: 'if me | %{user}さんが%{group}の%{meeting}という会議をキャンセルしました'
```

File: config/locales/ja.yml [hunk 34]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      join_meeting_subject: 'if me | %{user}は%{group}の%{meeting}という会議に参加しました'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      join_meeting_subject: 'if me | %{user}さんが%{group}の%{meeting}という会議に参加しました'
```

File: config/locales/ja.yml [hunk 35]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      join_meeting_body: '<p>%{meeting}について、%{link}！</p>'
+      reported_user_subject: 'if me | あなたはユーザーを報告しました。'
+      reported_user_body: 'あなたは%{reported_name}を報告しました。これは管理者によってレビューされます。'
+      reportee_user_subject: 'if me | あなたは報告されました'
+      reportee_user_body: 'あなたは報告されました。これは管理者によってレビューされます。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      reportee_user_body: 'あなたは報告されました。これは管理者にレビューされます。'
```

File: config/locales/ja.yml [hunk 36]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      update_meeting_subject: 'if me | %{user}は%{group}の%{meeting}という会議を更新しました'
+      new_meeting_subject: 'if me | %{user}は%{group}の%{meeting}という会議を作成しました'
+      meeting_body: '<p>%{subject}</p><p><i>%{description}</i></p><p><strong>場所：</strong>%{location}</p><p><strong>日付：</strong>%{date}</p><p><strong>時間：</strong>%{time}</p>'
+      new_meeting_link: '<p>詳しくは、%{link}！</p>'
+      update_meeting_link: '<p>詳しくは、%{link}！</p>'
+      remove_meeting_subject: 'if me | %{user}は%{group}の%{meeting}という会議をキャンセルしました'
+      join_meeting_subject: 'if me | %{user}は%{group}の%{meeting}という会議に参加しました'
+      join_meeting_body: '<p>%{meeting}について、%{link}！</p>'
+      reported_user_subject: 'if me | あなたはユーザーを報告しました。'
+      reported_user_body: 'あなたは%{reported_name}を報告しました。これは管理者によってレビューされます。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      reported_user_body: 'あなたは%{reported_name}を報告しました。これは管理者にレビューされます。'
```

File: config/locales/ja.yml [hunk 37]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      reportee_user_subject: 'if me | あなたは報告されました'
+      reportee_user_body: 'あなたは報告されました。これは管理者によってレビューされます。'
+      add_ban_subject: 'if me | あなたはアクセス禁止されています'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      add_ban_subject: 'if me | あなたはアクセス禁止にされました'
```

File: config/locales/ja.yml [hunk 38]
```diff
@@ -306,62 +306,62 @@ ja:
...
+      add_ban_body: 'あなたは%{code_of_conduct_link}に違反してアクセス禁止されています。'
+      remove_ban_subject: 'if me | アクセス禁止が外されました'
+      remove_ban_body: 'あなたは前に%{code_of_conduct_link}に違反してアクセス禁止されていました。もはや禁止されていません。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      remove_ban_body: 'あなたは%{code_of_conduct_link}に違反してアクセス禁止されていました。もう禁止されていません。'
```

**Comment by matcha440** [comment]
@nshki Thank you for the suggestions!  A lot of good feedback.  I'm so glad GitHub improved the suggested changes feature, very helpful! 👍

**Comment by nshki** [comment]
> @nshki Thank you for the suggestions! A lot of good feedback. I'm so glad GitHub improved the suggested changes feature, very helpful! 👍

Yes! Super happy about this feature too.

**Pull request merged**